lib/bulk_processor/back_end/dynosaur/tasks.rb

```diff
@@ -0,0 +1,2 @@
+require_relative 'process_csv_task'
+require_relative 'split_csv_task'
```

Comment:
The `dynosaur` version gets this init file, but the `active_job` version loaded its pair directly. We should be consistent for sake of readers.